tools/individual-pkg-gen/copy-files-removing-deps.ts

```diff
@@ -0,0 +1,201 @@
+          // devDependencies
+          const alphaDevDependencies: { [dep: string]: string } = {
+            // we need jsii-rosetta in the dependencies,
+            // otherwise the binary cannot be found when running the 'extract' script
+            'jsii-rosetta': jsiiRosettaVersion,
+          };
+          const devDependencies = phase1PackageJson.devDependencies || {};
+          for (const devDependency of Object.keys(devDependencies)) {
+            if (devDependency.startsWith('@aws-cdk/')) {
+              delete devDependencies[devDependency];
+            }
+            if (alphaPackages[devDependency]) {
+              alphaDevDependencies[alphaPackages[devDependency]] = pkg.version;
+            }
+          }
+          devDependencies['@aws-cdk/assert'] = pkg.version;
+          devDependencies['aws-cdk-lib'] = pkg.version;
+          devDependencies.constructs = '^10.0.0';
+          // we save the devDependencies in a temporary key in package.json
+          finalPackageJson.devDependencies = {
+            ...devDependencies,
+            ...alphaDevDependencies,
+            ...alphaDependencies,
+          };
+          phase1PackageJson.devDependencies = {
+            ...alphaDevDependencies,
+            ...alphaDependencies,
+          };
+
+          // peer dependencies
+          finalPackageJson.peerDependencies = {
+            'aws-cdk-lib': `^${pkg.version}`,
+            'constructs': '^10.0.0',
+            ...(Object.keys(alphaDependencies)
+              .reduce((acc, alphaDependency) => {
+                acc[alphaDependency] = `^${pkg.version}`;
+                return acc;
+              }, {} as { [dep: string]: string })),
+          };
+          phase1PackageJson.peerDependencies = undefined;
+
+          fileProcessed = true;
+          fs.writeJsonSync(destination, phase1PackageJson, { spaces: 2 });
+          fs.writeJsonSync(path.join(destDir, '_package.json'), finalPackageJson, { spaces: 2 });
+        }
+      } else if (sourceFileName === '.gitignore') {
+        // ignore everything, otherwise there are uncommitted files present in testing,
+        // because the module's .gitignore file has entries like !.eslintrc.js
+        fileProcessed = true;
+        const gitIgnoreContents = fs.readFileSync(source);
+        fs.outputFileSync(destination, Buffer.concat([gitIgnoreContents, Buffer.from('\n*\n')]));
+      } else if (sourceFileName === '.eslintrc.js') {
+        // Our ESLint configuration prevents importing libraries not in 'dependencies',
+        // which is against our philosophy for experimental modules (which use peerDependencies).
+        // Given that, remove that rule from the ESLint configuration of that module
+        fileProcessed = true;
+        const esLintRcContents = fs.readFileSync(source);
+        fs.outputFileSync(destination, Buffer.concat([esLintRcContents,
+          Buffer.from("\ndelete baseConfig.rules['import/no-extraneous-dependencies'];\n")]));
```

Comment:
see latest response to the comment in the previous round about this exemption.

Reply:
Done.